### PR TITLE
[11.0][MIG] base_tier_validation

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration to v11. Based on https://github.com/OCA/server-tools/pull/1203

### CODE REVIEW

File: base_tier_validation/__init__.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (MiquelRForgeFlow) [inline base_tier_validation/__init__.py]: in all `__init__` files you should remove full header

File: base_tier_validation/__manifest__.py [hunk 2]
```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Eficent Business and IT Consulting Services S.L.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Base Tier Validation",
     "summary": "Implement a validation process based on tiers.",
-    "version": "10.0.1.0.0",
+    "version": "11.0.1.0.0",
     "category": "Tools",
     "website": "https://github.com/OCA/server-tools",
```
Comment (MiquelRForgeFlow) [inline base_tier_validation/__manifest__.py]: -> `server-ux`

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: base_tier_validation/models/tier_validation.py [hunk 3]
```diff
@@ -90,7 +89,7 @@ def _compute_need_validation(self):
     def evaluate_tier(self, tier):
         try:
             res = safe_eval(tier.python_code, globals_dict={'rec': self})
-        except Exception, error:
+        except Exception as error:
```
Comment (MiquelRForgeFlow) [inline base_tier_validation/models/tier_validation.py]: you can remove `as error`

Comment (LoisRForgeFlow) [reply base_tier_validation/models/tier_validation.py]: I use it below

**Comment by lreficent** [comment]
WIP

File: base_tier_validation/__init__.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply base_tier_validation/__init__.py]: License must be kept: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: base_tier_validation/__manifest__.py (hunk 2 above)
Comment (LoisRForgeFlow) [reply base_tier_validation/__manifest__.py]: right!

File: base_tier_validation/models/tier_validation.py (hunk 3 above)
Comment (MiquelRForgeFlow) [reply base_tier_validation/models/tier_validation.py]: Ah, true, I checked too fast.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
Tested. Test Coverage is now very good.

**Pull request merged**